LibraryApp/jsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+  "compilerOptions": {
+    "baseUrl": "./src",
+    "paths": {
+      "@app": ["./src/app"],
+        "@assets": ["./src/app/assets"],
+        "@components": ["./src/app/components"],
+        "@config": ["./src/config"],
+        "@constants": ["./src/constants"],
+        "@redux": ["./src/redux"],
+        "@services": ["./src/services"],
+        "@utils": ["./src/utils"]
+    }
+  }
+}
```

Comment:
what is this file?

Reply:
this is for babel-module-resolver, in an attempt to achieve autocomplete of relative imports

Reply:
however, it only completes the folders, but not the files or other folders inside :(